packages/core/notifications/core-notifications-browser-internal/src/toasts/global_toast_list.tsx

```diff
@@ -56,7 +57,12 @@ export class GlobalToastList extends React.Component<Props, State> {
         })}
         data-test-subj="globalToastList"
         toasts={this.state.toasts.map(convertToEui)}
-        dismissToast={({ id }) => this.props.dismissToast(id)}
+        dismissToast={({ id }) => {
```

Comment:
nit: I prefer to preface events with `on`... e.g. `onDismissToast`.

Reply:
afaik, this is EUI api

Reply:
the name of the function in props could be changed potentially, but it was already named this prior to this PR. I could change it with the next batch of changes